src/components/button/button.fluent.scss

```diff
@@ -0,0 +1,143 @@
+@use '../../styles/utilities' as utils;
+
+@mixin theme() {
+    igc-button[variant='raised']::part(base),
+    igc-link-button[variant='raised']::part(base),
+    igc-button[variant='fab']::part(base),
+    igc-link-button[variant='fab']::part(base) {
+        box-shadow: var(--igc-elevation-0);
+    }
+
+    igc-button[size='small']::part(base),
+    igc-link-button[size='small']::part(base) {
+        border-radius: calc(var(--igc-border-radius) * 1);
+        --size: #{utils.rem(24px)};
+    }
+
+    igc-button[size='medium']::part(base),
+    igc-link-button[size='medium']::part(base) {
+        border-radius: calc(var(--igc-border-radius) * 1);
+        --size: #{utils.rem(28px)};
+    }
+
+    igc-button[size='large']::part(base),
+    igc-link-button[size='large']::part(base) {
+        border-radius: calc(var(--igc-border-radius) * 1);
+        --size: #{utils.rem(32px)};
+    }
+
+    // FLAT
+    igc-button[variant='flat']:not([disabled])::part(base),
+    igc-link-button[variant='flat']:not([disabled])::part(base) {
+        color: #{utils.color(gray, 900)};
+
+        &:hover {
+            color: #{utils.color(primary, 500)};
+
+            &::before {
+                background: transparent;
+            }
+        }
+
+        &:focus,
+        &:active {
+            color: #{utils.color(primary, 500)};
+
+            &::before {
+                opacity: .8;
+                background: #{utils.color(gray, 100)};
+            }
+        }
+    }
+
+    // OUTLINED
+    igc-button[variant='outlined']:not([disabled])::part(base),
+    igc-link-button[variant='outlined']:not([disabled])::part(base) {
+        color: #{utils.color(gray, 900)};
+        transition: color .15s ease-out;
+        box-shadow: 0 0 0 1px #{utils.color(gray, 800)};
+
+        &:hover {
+            color: #{utils.color(gray, 900)};
+
+            &::before {
+                opacity: 1;
+                background: #{utils.color(gray, 100)};
+            }
+        }
+
+        &:focus,
+        &:active {
+            color: #{utils.color(gray, 900)};
+
+            &::before {
+                opacity: 1;
```

Comment:
I believe these are no longer needed. Remove here and throughout.

Reply:
That opacity is required in order to override the one that comes from the material theme, otherwise, it's impossible to achieve the desired look and feel.